src/extensions/filter-control/bootstrap-table-filter-control.js

```diff
@@ -190,8 +191,20 @@ const UtilsFilterControl = {
           cookies.push(cookie)
         }
       })
-      return cookies
     }
+    if (foundLocalStorage) {
+      for (let i = 0; i < foundLocalStorage.length; i++) {
+        let cookie = foundLocalStorage.key(i)
+        if (/./.test(cookie)) {
+          cookie = cookie.split('.').pop()
+        }
+
+        if (!cookies.includes(cookie)) {
```

Comment:
Please make sure this is running properly in IE

Reply:
"includes" is not supported by IE (https://caniuse.com/#feat=array-includes)

Reply:
When we release a new version, we will convert the code of ES6 to ES5.

Reply:
Got it, thanks